docs/updating/update-to-35.md

```diff
@@ -138,7 +138,7 @@ export default class MyEditorUI extends EditorUI {
 **After**:
 
 ```js
-import { EditorUI } from 'ckeditor5/src/core';
+import { EditorUI } from 'ckeditor5/src/ui';
```

Comment:
Requires additional comment here. The previous path will work when upgrading to 35.0.0. But when upgrading to 36.0.0 it needs to be the proposed one.

Reply:
It makes sense to explain in this guide that this file's path changed in 36.0.0.

Reply:
I wouldn't. This is an example for dealing with `enableToolbarKeyboardFocus` which is unrelated.